mtp_noms_ops/assets-src/stylesheets/views/_checks.scss

```diff
@@ -165,3 +167,51 @@ table.mtp-table--review {
 .mtp-check-header__assigned-user {
   color: $red;
 }
+
+.mtp-check-cell__added-to-my-list::before {
```

Comment:
Should we add a `white-space: nowrap;` rule added (to match designs and make it easier to read). Maybe at least over certain screen widths